language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -164,6 +167,13 @@ public void sort(BallerinaCompletionContext context, T node, List<LSCompletionIt
                 CompletionItem variableCItem = VariableCompletionItemBuilder.build(varSymbol, varSymbol.getName().get(),
                         typeName);
                 completionItems.add(new SymbolCompletionItem(ctx, symbol, variableCItem));
+            } else if (symbol.kind() == PARAMETER) {
+                ParameterSymbol varSymbol = (ParameterSymbol) symbol;
+                TypeSymbol typeDesc = (varSymbol).typeDescriptor();
+                String typeName = typeDesc == null ? "" : CommonUtil.getModifiedTypeName(ctx, typeDesc);
```

Comment:
here the type desc could be null, instead what if we make the typeDescriptor API to return an Optional, which is to be consistent with the other APIs

Reply:
Removed this null check. There shouldn't be a case where the typedesc becomes null.